tmpl/wiki/index.md

```diff
@@ -20,6 +20,7 @@ MirageOS is in beta stage and the infrastructure you see here is self-hosting. T
 <ul>
 <li><a href="/wiki/install">Installation</a></li>
 <li><a href="/wiki/hello-world">Hello Mirage World</a></li>
+<li><a href="/wiki/learning">Learning about Mirage</a></li>
```

Comment:
I wonder whether this should be listed above Hello Mirage World (but I don't feel strongly about it).